community/security/src/test/java/org/neo4j/server/security/auth/FileUserRepositoryTest.java

```diff
@@ -157,12 +157,14 @@ public void shouldNotAllowComplexNames() throws Exception
                 "The provided username is empty." );
         assertException( () -> users.assertValidUsername( "" ), InvalidArgumentsException.class,
                 "The provided username is empty." );
-        assertException( () -> users.assertValidUsername( ":" ), InvalidArgumentsException.class,
-                "Username ':' contains illegal characters. Use simple ascii characters and numbers." );
+        assertException( () -> users.assertValidUsername( "," ), InvalidArgumentsException.class,
+                "Username ',' contains illegal characters. Use simple ascii characters and numbers." );
```

Comment:
"... Use simple ascii characters and numbers." <- this part of the error message should maybe be changed?